### PR TITLE
New early math lesson

### PR DESCRIPTION
**Pull request by retabak**
entered the world of.... fractions !

### CODE REVIEW

File: lessons/Early-Math/inverse-relationships/langs/en-us/pages/divide-first-or-left-to-right.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+= Divide First... or Solve Left-to-Right?
```
Comment (schanzer) [inline lessons/Early-Math/inverse-relationships/langs/en-us/pages/divide-first-or-left-to-right.adoc:1]: This is _excellent_. Really nicely done!

I fixed a small formatting issue

**Review by schanzer — CHANGES_REQUESTED** [review]
This looks _really_ awesome. I think you'll likely learn a lot by testing it out in the 1-day PD, so I'm mostly recommending small changes. The one major issue is the workbook page that I called out -- happy to chat about it by phone if you want.

File: lessons/Early-Math/inverse-relationships/langs/en-us/pages/discover-inverse.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+| @showsoln{(coe '(/ 30 5))} | @showsoln{(coe  '(* 30 1/5))}
+| @ifsoln{@math{30 \div 5 = 6}} | @ifsoln{@math{30 \times \frac{1}{5} = 6}}
+|===
+
+
+
+@n Examine the Circles of Evaluation below.
```
[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Early-Math/inverse-relationships/langs/en-us/pages/discover-inverse.adoc:43]: omg, let's please discuss. I have no idea how to fix it! I totally leaned hard on "examine" when "analyze" was firmly rejected by you and Flannery.

File: lessons/Early-Math/inverse-relationships/langs/en-us/pages/discover-inverse.adoc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+|@show{(coe '(/ 20 1/4))}  | @show{(coe  '(* 20 4))}
+| @ifnotsoln{@math{20 \div \frac{1}{4} = ?}} @ifsoln{@math{20 \div \frac{1}{4} = 80}}  | @ifnotsoln{@math{20 \times 4 = ?}} @ifsoln{@math{20 \times 4 = 80}}
+|===
+
+How is this pair of Circles similar to the first pair on this page? How is it different? @fitb{}{@ifsoln{The same values are being used, but this time we are dividing by the fraction,}}
```
[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Early-Math/inverse-relationships/langs/en-us/pages/discover-inverse.adoc:53]: funny, I originally had something else then decided it should be the same to help students see the difference. But I guess it didn't have that effect?

File: lessons/Early-Math/inverse-relationships/langs/en-us/pages/discover-inverse.adoc [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+@fitb{}{}
+
+@vspace{1ex}
+
+Did you know that dividing by @math{4} is the same as multiplying by @math{\frac{1}{4}} ? Let's explore other division expressions and see if we can transform them similarly.
+
+@vspace{1ex}
+
+Below, draw another pair of Circles tp demonstrates how _dividing by a whole number_ produces the same result as _multiplying by its reciprocal._
```
[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Early-Math/inverse-relationships/langs/en-us/pages/discover-inverse.adoc:31]: good call.

**Pull request closed without merging**